front-end/hot-dog/src/CustomerComponents/CustomerMenu.js

```diff
@@ -7,12 +7,36 @@ class CustomerMenu extends React.Component{
         this.state = {tableDataisFetched: false};
     }
 
+    submitOrder = () => {
+        // create JSON object for back-end API
+        let order = {}
+        for(let key in this.state){
+            order[key] = this.state[key]
+        }
+        let orderJSON = {
+            "order": order,
+            "cartID":this.props.cartID,
+            "userID":2
+        }
+
+        fetch(`http://localhost:5000/customer/order/`, {
+            method:'PUT',
+            body:orderJSON,
```

Comment:
Is this a duplicate `body`? It's okay if the code is working for now, might want to delete it in future versions